getting-started/templates/systemlink-values.yaml

```diff
@@ -739,6 +773,33 @@ dataframeservice:
       ##
       passwordKey: "password"
 
+  ## The timeout for interactive metadata queries ([d.]hh:mm[:ss] format).
+  interactiveMetadataQueryTimeout: "00:00:10"
+
+  ## Configuration of the MongoDB database used for metadata storage.
+  database:
+    ## Custom properties to index. Each index will allow queries that filter on the custom property
```

Comment:
Suggestion: 'Contains the custom properties to index. Each index makes certain queries more efficient. These queries include those that filter on the custom property and queries that use the `ROWS_MODIFIED_AT` property to order data.'